paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreen.kt

```diff
@@ -35,6 +57,21 @@ internal fun UpdateCardScreenBody(
             style = MaterialTheme.typography.h2
         )
 
+        StripeThemeForLink {
+            CardDetailsEditUI(
+                editCardDetailsInteractor = interactor,
+            )
+        }
+
+        if (isDefault) {
+            Text(
+                modifier = Modifier.padding(top = 8.dp),
+                text = stringResource(R.string.stripe_link_update_card_default_card),
+                style = MaterialTheme.typography.subtitle2,
+                color = MaterialTheme.colors.onSecondary
+            )
+        }
```

Comment:
tried to match styling with what iOS have. See Android screenshots below and compare with this on iOS: 
<img width="424" alt="image" src="https://github.com/user-attachments/assets/ccf5cbd5-d131-458c-b235-8e634d6f3045" />